src/pages/index.js

```diff
@@ -67,17 +67,19 @@ function Home() {
         <video
           playsInline="playsinline"
           autoPlay="autoplay"
+          disablePictureInPicture
           muted="muted"
           loop="loop"
-          poster="img/fallback.jpg"
+          poster="img/fallback.avif"
         >
-          <source src="video/video.mp4" type="video/mp4" />
+          <source src="video/video.webm" type="video/webm" />
```

Comment:
WebM file was encoded with VP9. As far I can tell, AV1 is widely supported only on devices with a dedicated hardware decoder (and sometimes require more setup), so it might be too early for that. I can re-add mp4 for fallback but it'll need to be compressed to match webm's file size

Reply:
Yeah that sounds good 👍 I was assuming VP9, AV1 is a lot newer than I thought